tests/fake-nested-templates/cookiecutter.json

```diff
@@ -1,5 +1,14 @@
 {
-  "template": [
-    "fake-project (fake-project)"
-  ]
+  "templates": {
+    "fake-project": {
+      "path": "./fake-project",
+      "title": "A Fake Project",
+      "description": "A cookiecutter template for a project"
+    },
+    "fake-package": {
+      "path": "./fake-package",
+      "title": "A Fake Package",
+      "description": "A cookiecutter template for a package"
+    }
```

Comment:
Would it make sense to make the `templates` keys the paths themselves?

```
{
  "./fake-project": {...},
  "./fake-package": {...}
}
```

Reply:
I decided against it to allow, in the future, the path to point to another cookiecutter template. (i.e.: gh:foo/bar)